webapp/src/Entity/Problem.php

```diff
@@ -30,7 +30,8 @@
 #[ORM\Index(columns: ['special_compare'], name: 'special_compare')]
 #[ORM\HasLifecycleCallbacks]
 #[UniqueEntity(fields: 'externalid')]
-class Problem extends BaseApiEntity
+class Problem extends BaseApiEntity implements HasExternalIdInterface,
+    ExternalIdFromInternalIdInterface, PrefixedExternalIdInterface
```

Comment:
```suggestion
    ExternalIdFromInternalIdInterface, 
    PrefixedExternalIdInterface
```

I'm not sure if this is something we want in PHPCS.